### PR TITLE
COMMERCE-9599

### PR DESCRIPTION
**Pull request by alessiorendina**
https://liferay.atlassian.net/browse/COMMERCE-9599
Based on #4245 

@veroglez I re-organized the commits in order to simplify the code review. Thank you for your work!

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:sf

**Review by andrea-ale-sbarra — COMMENTED** [review]
Please have a look at my comments.
Thank you!

**Comment by alessiorendina** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItem.js [hunk 1]
```diff
@@ -169,34 +169,70 @@ function CartItem({
...
-			>
-				{!!adaptiveMediaImageHTMLTag && (
+			{Liferay.FeatureFlags['COMMERCE-8715'] ? (
+				<div className="mini-cart-item-details position-relative">
+					<a
+						className="h-100 mini-cart-item-anchor position-absolute w-100"
+						data-senna-off="true"
+						href={productPageUrl}
+					>
+						<span className="sr-only">
```
[the remark this answers is not in the document]
Comment (alessiorendina) [reply modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItem.js:192]: You're right. Fixed!

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItemViews/ItemInfoView.js [hunk 2]
```diff
@@ -19,15 +20,56 @@ function ItemInfoViewOptions({options}) {
...
+
+						return (
+							<div className="child-item" key={index}>
+								<span>
+									<>
+										{quantity} &times; {name}
+									</>
+									<> {skuUnitOfMeasure?.key || ''}</>
+								</span>
+							</div>
```
[the remark this answers is not in the document]
Comment (alessiorendina) [reply modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItemViews/ItemInfoView.js:55]: This is part of the development that @smottal did in order to show the unit of measure in the mini cart row

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItem.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+					<ClayButtonWithIcon
+						aria-label={backLabel}
+						displayType="unstyled"
+						onClick={() => setEditedItem(null)}
+						symbol="angle-left"
+						title={backLabel}
+					/>
+
+					<span className="font-weight-bold ml-2 text-5">
+						{editedItem.name}
```
[the remark this answers is not in the document]
Comment (alessiorendina) [reply modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItem.js:177]: I'll add it, thanks!

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by victoroscuro** [comment]
Tests failing on the relevant seem not to be related to the PR

**Comment by victoroscuro** [comment]
ci:forward:force

**Comment by riccardo-alberti** [comment]
Manuelly forwarded here: https://github.com/brianchandotcom/liferay-portal/pull/141635

**Pull request closed without merging**